test/Microsoft.AspNetCore.FunctionalTests/WebHostFunctionalTests.cs

```diff
@@ -4,31 +4,20 @@
 using System;
 using System.Collections.Generic;
 using System.IO;
-using System.Linq;
 using System.Net.Http;
 using System.Threading.Tasks;
-using Microsoft.AspNetCore.Hosting;
-using Microsoft.AspNetCore.Hosting.Server.Features;
 using Microsoft.AspNetCore.Http;
 using Microsoft.AspNetCore.Server.IntegrationTesting;
 using Microsoft.AspNetCore.Testing.xunit;
 using Microsoft.Extensions.Logging;
 using Microsoft.Extensions.Logging.Testing;
 using Xunit;
-using Xunit.Abstractions;
 using Xunit.Sdk;
 
 namespace Microsoft.AspNetCore.Tests
 {
     public class WebHostFunctionalTests : LoggedTest
     {
-        private readonly string _testSitesPath;
-
-        public WebHostFunctionalTests(ITestOutputHelper output) : base(output)
-        {
-            _testSitesPath = GetTestSitesPath();
-        }
-
```

Comment:
Note that there are two other tests `CreateDefaultBuilder_InitializeWithDefaults` and `CreateDefaultBuilderOfT_InitializeWithDefaults` that uses a hardcoded port 5002 to ensure the default app settings are loaded. I'll see if I can use another setting instead since hard coding ports is a bad pattern.